src/core/public/index.scss

```diff
@@ -7,5 +7,54 @@
 // Mixins provide generic code expansion through helpers
 @import '@elastic/eui/src/global_styling/mixins/index';
```

Comment:
Just trying to understand the SSS structure.

Does this file not behave similarly to the other JS imported SASS files where it automatically compiles with the `styling_constants`?

Reply:
This file is compiled by both the new and legacy system so it must import these directly.